lib/php_serialize.rb

```diff
@@ -210,6 +210,40 @@ def string.read_until(char)
 		end
 	end
 
+  ##
+  # Tests if an input is valid PHP serialized string.
+  #
+  # Checks if a string is serialized using quick string manipulation
+  # to throw out obviously incorrect strings.
+  #
+  # http://stackoverflow.com/questions/1369936/check-to-see-if-a-string-is-serialized
+  def PHP.serialized?(string)
+    # If it isn't a string, it isn't serialized
+    return false unless string.instance_of? String
+
+    # Remove any trailing whitespace
+    string.chomp!
```

Comment:
Mutating the argument is a bit strange, seems like we should make a new variable here. Or alternatively, should we be chomping at all? If a end-user adds whitespace, it's up to them to delete it.